src/test/java/com/nttdata/nttdatanotificationservice/controller/SplunkNotificationControlllerTest.java

```diff
@@ -58,8 +58,7 @@ void initialize() throws NoSuchMethodException, InvocationTargetException, Illeg
     @DisplayName("Success - SplunkNotificationController")
     @Test
     void testSuccess() {
-        Gson gson = new Gson();
-        SplunkAlert splunkAlert = gson.fromJson(splunkAlertJson, SplunkAlert.class);
+        String splunkAlert = splunkAlertJson;
 
         when(webHookService.postMessage(any(), any())).thenReturn(new ResponseEntity<>(
                 "We good", HttpStatus.OK));
```

Comment:
out of scope but if this is going to prod, we need to change this and anything similar to it.